workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/JiraRestController.java

```diff
@@ -235,11 +417,23 @@ HttpHeaders.AUTHORIZATION, _getCredentials()
 		return null;
 	}
 
-	private boolean _isPartner() {
-		return true;
+	private boolean _isPartner(Jwt jwt) {
```

Comment:
can you rename this method? it's no longer looking for just partners. Maybe hasPartnerPublishDatePermission or hasEarlyPublishAccess

Reply:
Done!